README.md

```diff
@@ -124,11 +124,65 @@ caching by way of expiry or ETags, I recommend using `faraday-http-cache` for
 your clients and setting `memoize` to `false`. Remote Record will eventually
 gain support for caching.
 
+### `remote_all`
+
+If you're able to fetch multiple records at once from the API, implement the
+`self.all` method on your remote record class. This should return an array of
+hashes that can be used to initialize a set of references.
+
+This can optionally take a block
+for authorization - note that it won't use the auth you've configured and that
+you'll always have to supply that inline. For example:
+
+```ruby
+module RemoteRecord
+  module GitHub
+    # :nodoc:
+    class User < RemoteRecord::Base
+      def get
+        client.user(remote_resource_id)
+      end
+
+      def self.all
+        Octokit::Client.new(access_token: yield).users
+      end
+
+      private
+
+      def client
+        Octokit::Client.new(access_token: authorization)
+      end
+    end
+  end
+end
+```
+
+Now you can call `remote_all` on remote reference classes that use
+`RemoteRecord::GitHub::User`, like this:
+
+```ruby
+GitHub::UserReference.remote_all { GITHUB_PERSONAL_ACCESS_TOKEN }
+```
+
+### `initial_attrs`
```

Comment:
maybe just call this `attrs`

Reply:
`initial` is important — changes to this value aren't reflected inside the `instance`.